src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -18,18 +17,34 @@ enum TABS {
   LIVE_SCHEDULE = 2,
 }
 
+const DOCUMENT_ID = "__name__";
+
+// Maybe move this to  utils?
+const filterUniqueKeys = (userId: string, index: number, arr: string[]) =>
+  arr.indexOf(userId) === index;
```

Comment:
Agree on moving this to `utils`. I tend to name my filters `fooprefixFilter`, so in this case: `uniqueKeysFilter`. So that when it is written out it will be `foo.filter(uniqueKeysFilter)`